NebulaClient/GameLogic/PlayerManager.cs

```diff
@@ -1,79 +1,84 @@
 ﻿using NebulaModel.Logger;
 using System.Collections.Generic;
+using UnityEngine;
+using NebulaModel.DataStructures;
 
 namespace NebulaClient.GameLogic
 {
-    public class PlayerManager
-    {
-        Dictionary<ushort, Player> remotePlayers;
-        Dictionary<ushort, RemotePlayerModel> remotePlayerModels;
+	public class PlayerManager
+	{
+		Dictionary<ushort, Player> remotePlayers;
+		Dictionary<ushort, RemotePlayerModel> remotePlayerModels;
```

Comment:
@hubastard Note to self, I'll need to add a `editorconfig` file with the code formatting rules of my project to prevent issues like different tabs identation that we see here.

Reply:
So, i change the tabs to spaces right?

Reply:
Yeah, but if you merge master in your branch you should be able to now click the 'clean' button in a file to reformat it using the new .editorconfig file.

Reply:
Do you mean right-clicking solution ==> Clean Solution?